storage-hbase/src/main/java/org/apache/kylin/storage/hbase/steps/HBaseJobSteps.java

```diff
@@ -72,6 +80,26 @@ public HadoopShellExecutable createCreateHTableStep(String jobId, CuboidModeEnum
         return createHtableStep;
     }
 
+    public AbstractExecutable createDistcpHFileStep(String jobId){
+        String inputPath = getRealizationRootPath(jobId) + "/hfile";
+        MapReduceExecutable createHFilesStep = new MapReduceExecutable();
+        createHFilesStep.setName(ExecutableConstants.STEP_NAME_HFILE_DISTCP);
+        StringBuilder cmd = new StringBuilder();
+
+        appendMapReduceParameters(cmd);
+        appendExecCmdParameters(cmd, BatchConstants.ARG_CUBE_NAME, seg.getRealization().getName());
+//        appendExecCmdParameters(cmd, BatchConstants.ARG_PARTITION, getRowkeyDistributionOutputPath(jobId) + "/part-r-00000_hfile");
```

Comment:
remove commented code if not needed , will help in code readability going forward.